lib/Terrain.cpp

```diff
@@ -397,30 +397,6 @@ TerrainType::TerrainType(const std::string& _name):
 	transitionRequired(false)
 {
 }
-
-TerrainType& TerrainType::operator=(const TerrainType & other)
```

Comment:
Do not need it anymore? Or is it autogenerated thing?

Reply:
Yes, compiler will generate exactly the same, as here simply all fields are copied. Besides, classic implementation of = must have protection from assigning an object to itself.